loginCompose/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
```

Comment:
Split strings for this module that only it will use, so no things like "app_name"

Reply:
moved the strings to ui core instead. Should I have a string values specificaly just for this module instead though?